### DISCUSSION TITLE
[QA] 서로 버그 리포트해주기??

### DISCUSSION THREAD

**Discussion by Hyevvy** [prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung#240] [General]
어느정도 기능 구현이 끝난 거 같아서 서로 페이지 보면서 하루정도는 피드백하고 반영하는 시간을 가져도 좋을 거 같아요 😳

ex) 포스트 개수가 화면에 뿌려지는 개수 미만일 때는 더보기가 안 보이게 수정하기

**Comment by YuHyun-P** [comment]
발표 자료를 위한 기능 설명?도 적으면 좋을 것 같아요!

**Reply by Hyevvy** [reply]
좋아용 각자 담당한 페이지마다 적으면 되겠네용!1

**Reply by YuHyun-P** [reply]
👉🏼 페이지 별 기능 [Wiki](https://github.com/prgrms-fe-devcourse/FEDC2_2JAVATAYO_Dongyoung/wiki/2JAVATAYO-%ED%8E%98%EC%9D%B4%EC%A7%80-%EB%B3%84-%EA%B8%B0%EB%8A%A5)

**Comment by Hyevvy** [comment]
현재 버그 리포트 가능한 페이지 : 메인(카드 + 더보기) + 유저검색 제외
